lib/java/src/org/apache/thrift/protocol/TBinaryProtocol.java

```diff
@@ -279,6 +280,10 @@ public void readFieldEnd() throws TException {}
   @Override
   public TMap readMapBegin() throws TException {
     TMap map = new TMap(readByte(), readByte(), readI32());
+
+    //
```

Comment:
there are quite a few emtpy comment lines in here, is there sth missing?

Reply:
This is a redundant comment.

Reply:
This is a redundant comment.